tests/test_exporters.py

```diff
@@ -485,6 +497,25 @@ def test_nonstring_types_item(self):
         self.assertEqual(exported, [item])
 
 
+class JsonItemExporterToBytesTest(BaseItemExporterTest):
```

Comment:
Here you're running 5 more tests from the base class - is it intended? If you don't want to do this, you can add another test case to JsonItemExporterTest and use `ie = self._get_exporter(encoding='latin-1')` instead of self.ie

Reply:
I didn't mean to add the extra 5 tests, no; I'll revisit trying to use `ie` directly but the methods to handle it as an attribute and get the output were very helpful in keeping the new test-case small and tidy. :/